### ISSUE TITLE
Updated Angular shim code, can you please rebuild?

### ISSUE THREAD

**Issue by jesusfreak3** [rvl/bower-angular-component-router#1]
Hello,  I made a change in the shim file in the angular repo and it was merged.  Do you think you can just re build your generated js files?

**Comment by rvl** [comment]
Sure can, but has your change been released on npm? If so, which version?

Also are you able to provide any information/summary about what is going on with the angular router mess?

Cheers.

**Comment by jesusfreak3** [comment]
Hey, 

Good question, I am not sure if it was added in the NPM.  My change just fixed a bug when using the shim, if you had an anchor that was not using nglink, nothing would happen.  So added the case to match like Angular 1 where adding target="_blank/_self, etc" the nglink would pass it through.  I know there were a few other minor fixes since your current version as well.

My commit merged

https://github.com/angular/angular/commit/be48ba1b06c2b6450b8d8003e489b5d8c16eecaa

Thanks

**Comment by rvl** [comment]
I'm just trying to track the npm releases here. Are you able to bug them to get a release of angular_1_router at version [2.0.0-rc.3](https://github.com/angular/angular/releases/tag/2.0.0-rc.3) onto npm?

**Comment by thrilleratwork** [comment]
I believe this was merged into 2.0.0-rc.4

**Comment by jesusfreak3** [comment]
Yes I see my changes are in the rc.4 tag, so we should be good to rebuild now.

**Comment by rvl** [comment]
Ideally, I need this release to be uploaded to npm. That is where the bower package comes from. If you check `npm info @angular/router`, what is the latest version?
